src/plugins/telemetry/schema/oss_plugins.json

```diff
@@ -5635,6 +5635,137 @@
             }
           }
         },
+        "slo": {
+          "properties": {
+            "appId": {
+              "type": "keyword",
+              "_meta": {
+                "description": "The application being tracked"
+              }
+            },
+            "viewId": {
+              "type": "keyword",
+              "_meta": {
+                "description": "Always `main`"
+              }
+            },
+            "clicks_total": {
+              "type": "long",
+              "_meta": {
+                "description": "General number of clicks in the application since we started counting them"
```

Comment:
```suggestion
                "description": "Total number of clicks in the application since we started counting them"
```
"General" is too vague, one doesn't know if it's an average, weighted average or something else.

Reply:
@TinaHeiligers Thanks I [updated the descriptions](https://github.com/elastic/kibana/pull/177937/commits/fedc3b9ea6ce7b68b68dcda25e58f36ac6cd0f2e) accordingly, replacing `General` with `Total`

Since these descriptions were generated after I ran `node scripts/telemetry_check --fix`, I updated the default description in the [common schema](https://github.com/elastic/kibana/pull/177937/commits/456501b609ee9a7b32f1496a6e5afbb92712810f).

Btw do you know where can I check in Kibana the generated telemetry. [Here's](https://github.com/elastic/kibana/pull/132006#issuecomment-1149553940) how I used to do it in the past, but it doesn't work this way now

Reply:
> https://github.com/elastic/kibana/pull/132006#issuecomment-1149553940 how I used to do it in the past, but it doesn't work this way now

oh, that's news to me. What doesn't work? Is the button missing, are your fields not there?

Reply:
> Stack Management > Advanced Settings > bottom of the page Usage Data

I don't see anything about `Usage Data` under Advanced Settings. How can I test the telemetry that will be sent while I navigate to the SLO app?

Reply:
Btw after me applying the change regarding Renaming `General` to `Total` CI complains and wants me to update all descriptions. I did the change in this [commit](https://github.com/elastic/kibana/pull/177937/commits/e88ce4e6c7579eaaadfa18eb32bd17ad756fbaf4) (not manual, I simply ran `node scripts/telemetry_check --fix `). Is it fine or do you want me to revert back to `General`?

Reply:
The Usage collection section moved to the "Global Settings" tab in Advanced Settings:

<img width="446" alt="Screenshot 2024-03-12 at 14 34 17" src="https://github.com/elastic/kibana/assets/11909450/681611af-f8b6-4707-8436-801a7deae648">

<img width="821" alt="Screenshot 2024-03-12 at 14 31 28" src="https://github.com/elastic/kibana/assets/11909450/0f8eaf74-3672-47aa-aff1-0ff5b7bb5ae7">

<img width="705" alt="Screenshot 2024-03-12 at 14 31 57" src="https://github.com/elastic/kibana/assets/11909450/199f53c1-630a-4f82-9c00-13982962130a">

Reply:
Awesome! That's what I was searching for, thanks!

Regarding my question [here](https://github.com/elastic/kibana/pull/177937#discussion_r1522136095), is it fine?

Reply:
About those descriptions: It looks like other applications also use the "General" term. Maybe it's best to keep them as you had.

Up to you thought, these are for internal use.

Telemetry related changes LGTM

Reply:
Agree, I kept them as before. Thanks!